vitals/BatteryFactorAccumulator.java

```diff
@@ -0,0 +1,89 @@
+package vitals;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BatteryFactorAccumulator {
+	
+	public static String language;
+	static List<String> lang_types = Arrays.asList("EN", "DE");
+	static Map<String,Float> VitalFactorExceedMinLimit=new HashMap<>();
+	static Map<String,Float> VitalFactorReachHighBreach=new HashMap<>();
+	static Map<String,Float> VitalFactorReachLowBreach=new HashMap<>();
+	static Map<String,Float> VitalFactorExceedMaxLimit=new HashMap<>();
+	
+
+    public static void showMsg(String msg)
+    {
+      System.out.println(msg);
+    }
+    
+    public static void displayMaximumAlert(String vitalparam)

```

Comment:
function name is display ...but checks the condition as well

string construction duplicated

Reply:
changed method name to checkAndPrintMaximumAlert